.github/workflows/ci.yml

```diff
@@ -244,8 +268,9 @@ jobs:
         cargo-clippy,
         cargo-fmt-check,
         cargo-run-e2e-test,
+        cargo-test-lib-std,
         cargo-test-workspace,
-        build-sway-lib-std,
```

Comment:
I also noticed that this `build-sway-lib-std` `need` entry was duplicated and that we were missing `cargo-test-lib-std`! I've fixed this up while adding the new `cargo-unused-deps-check`. Same goes for the other publish jobs.